package/.eslintrc.js

```diff
@@ -1,51 +0,0 @@
-module.exports = {
```

Comment:
@rgomezp Are we removing the ESLint?

If we abandon ESlint, it's going to be much harder to review PRs and keep a code pattern, avoiding common mistakes.

Reply:
That's a good point. Looks like I threw the baby out with the bathwater

Reply:
Let's recover that baby then hahahah 😂